mysql/check.py

```diff
@@ -528,7 +528,11 @@ def _collect_metrics(self, host, db, tags, options, queries):
 
         if _is_affirmative(options.get('replication', False)):
             # Get replica stats
-            results.update(self._get_replica_stats(db))
+            is_mariadb = self._get_is_mariadb(db, host)
+            replication_channel = options.get('replication_channel', False)
```

Comment:
I think I would rather have:
```
replication_channel = options.get('replication_channel')
```

`replication_channel = None` would also be evaluated to `False` so that should be enough.